src/api/postRfc003Swap.ts

```diff
@@ -1,9 +1,24 @@
 import axios from "axios";
 import update from "immutability-helper";
 import { Rfc003Params } from "../forms/Rfc003ParamsForm";
-import { Swap } from "../forms/SwapForm";
+import { Swap, SwapValue } from "../forms/SwapForm";
 import { relativeMinutesToTimestamp } from "../time";
 import apiEndpoint from "./apiEndpoint";
+import { toSmallestUnit } from "./unit";
+
+function adjustAssetUnit(value: SwapValue) {
+  const asset = {
```

Comment:
📣 Please add a unit test that checks that the unit and amount sent to `comit_node` is actually in Satoshi.